game/src/Game/GameSession/ProvinceOverviewPanel.gd

```diff
@@ -307,23 +308,30 @@ func _update_info() -> void:
 		# Statistics
 		if _rgo_icon:
 			_rgo_icon.set_icon_index(_province_info.get(_province_info_rgo_icon_key, -1) + 2)
+			print(_province_info.get(_province_info_rgo_name_key))
 
 		if _rgo_produced_label:
 			_rgo_produced_label.text = GUINode.float_to_string_dp(_province_info.get(_province_info_rgo_output_quantity_yesterday_key, 0), 3)
-
+			print()
```

Comment:
What does this do?

Reply:
it was some old test code to see if stuff was being passed correctly removed it now